### PR TITLE
Wilsonj806/containerization instructions update

### PR DESCRIPTION
**Pull request by wilsonj806**

### CODE REVIEW

**Review by justintmckibben — APPROVED** [review]
Thanks for doing this!!

File: README.md [hunk 1]
```diff
@@ -109,24 +108,24 @@ https://docs.google.com/document/d/1UPNCwjWS_T7XT5AXsewphu6NvNdV7TQLSJub-RBRAG0/
...
-cd api && docker-compose up
-# navigate to localhost:3000
+cd <scl-be directory> && docker-compose up
 ```
 accessing rail CLI after `docker-compose` is running.
 
 ```
-docker exec -it api_web_1 bash
-# now inside docker container example of commands that can be run
+# Replace <scl-be directory name> with the name of the direcotry that you cloned the repo into
```
Comment (jacoblb64) [inline README.md:120]: maybe add a comment that a user can confirm the image name with `docker-compose ps`?

File: config/database.yml [hunk 2]
```diff
@@ -29,6 +29,8 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
+  # NOTE: You will need to uncomment  and use the `username` value that corresponds with
+  # ...the POSTGRES_USER value in docker-compose.yml
```
Comment (jacoblb64) [inline config/database.yml:33]: just to confirm -- should these be uncommented for local or docker running of the database?

**Review by jacoblb64 — APPROVED** [review]
Thank you so much for doing this! Left some small possibly clarifying comments if you want, but they're completely optional.

File: config/database.yml (hunk 2 above)
Comment (wilsonj806) [reply config/database.yml]: Oh yeah, I forgot to specify that. Updated with that in mind and your above comment about `docker-compose ps`

Comment (jacoblb64) [reply config/database.yml]: Awesome, thanks for taking care of this again and and clarifying the instructions!

**Pull request merged**